curriculum/challenges/english/blocks/lecture-best-practices-for-styling-forms/672bca660aa9f9ce9b2b2787.md

```diff
@@ -9,7 +9,7 @@ dashedName: what-are-common-issues-when-styling-special-input-elements
 
 Let's learn about some of the common issues when trying to style special input elements like the `datetime-local` and `color` inputs.
 
-These special types of inputs rely on complex pseudo-elements to create things like the date and color pickers. This presents a significant challenge for styling these inputs. One challenge is that the default styling is entirely browser-dependent, so the CSS you write to make the picker look the way intend may be entirely different on another browser. 
+These special types of inputs rely on complex pseudo-elements to create things like the date and color pickers. This presents a significant challenge for styling these inputs. One challenge is that the default styling is entirely browser-dependent, so the CSS you write to make the picker look the way you intend may be entirely different on another browser. 
```

Comment:
Thanks @bdtripp.

While we're at this, I think we could rephrase the sentence. It feels a bit clunky to me.

```suggestion
These special types of inputs rely on complex pseudo-elements to create things like the date and color pickers. This presents a significant challenge for styling these inputs. One challenge is that, because the default styling depends entirely on the browser, CSS that makes the picker look right in one browser may produce a very different result in another.
```